packages/cli/src/util/dev/router.ts

```diff
@@ -58,7 +58,7 @@ export async function devRouter(
 ): Promise<RouteResult> {
   let result: RouteResult | undefined;
   let { pathname: reqPathname, search: reqSearch } = url.parse(reqUrl);
-  reqPathname ??= '/';
+  reqPathname = reqPathname || '/';
```

Comment:
I don't understand this change. How did we regress in the syntax we support?

Reply:
@Ethan-Arrowood  Previously, TS was downleveling and then ncc would bundle it.

Since TS is no longer downleveling, ncc is choking because its very outdated https://github.com/vercel/vercel/blob/dde2b9b34f262addc89eea599b3bab047e28f4d7/packages/cli/package.json#L95